packages/contracts-bedrock/scripts/checks/interfaces/main_test.go

```diff
@@ -0,0 +1,295 @@
+package main
```

Comment:
Is this tested somewhere in ci?

Reply:
Ah good point, I don't think that it is. Mind if I add this in a separate PR that includes testing in CI for all of the other Go checks?

Reply:
That's OK.